.github/workflows/ci.yml

```diff
@@ -238,96 +251,18 @@ jobs:
-          tags: |
-            type=semver,pattern={{version}}
-            type=edge
-            type=ref,event=pr
-            type=ref,event=branch,suffix=-rc,enable=${{ startsWith(github.ref, 'refs/heads/release') }}
-
-      - name: Build NGF Docker Image
-        uses: docker/build-push-action@2cdde995de11925a030ce8070c3d77a52ffcf1c0 # v5.3.0
-        with:
-          file: build/Dockerfile
-          tags: ${{ steps.ngf-meta.outputs.tags }}
-          context: "."
-          target: goreleaser
-          load: true
-          cache-from: type=gha,scope=ngf
-          pull: true
-
-      - name: Build NGINX Docker Image
-        uses: docker/build-push-action@2cdde995de11925a030ce8070c3d77a52ffcf1c0 # v5.3.0
-        with:
-          file: build/Dockerfile.nginx
-          tags: ${{ steps.nginx-meta.outputs.tags }}
-          context: "."
-          load: true
-          cache-from: type=gha,scope=nginx
-          pull: true
-          build-args: |
-            NJS_DIR=internal/mode/static/nginx/modules/src
-            NGINX_CONF_DIR=internal/mode/static/nginx/conf
-            BUILD_AGENT=gha
-
-      - name: Deploy Kubernetes
-        id: k8s
-        run: |
-          kube_config=${{ github.workspace }}/kube-${{ github.run_id }}-helm
-          make create-kind-cluster KIND_KUBE_CONFIG=${kube_config}
-          echo "KUBECONFIG=${kube_config}" >> "$GITHUB_ENV"
-          kind load docker-image ghcr.io/nginxinc/nginx-gateway-fabric:${{ steps.ngf-meta.outputs.version }} ghcr.io/nginxinc/nginx-gateway-fabric/nginx:${{ steps.nginx-meta.outputs.version }}
-          kubectl kustomize config/crd/gateway-api/standard | kubectl apply -f -
-
-      - name: Install Chart
-        run: >
-          helm install
-          helm-$(echo ${{ steps.ngf-meta.outputs.version }} | tr '.' '-')
-          .
-          --wait
-          --create-namespace
-          --set nginxGateway.image.repository=ghcr.io/nginxinc/nginx-gateway-fabric
-          --set nginxGateway.image.tag=${{ steps.ngf-meta.outputs.version }}
-          --set nginxGateway.image.pullPolicy=Never
-          --set nginxGateway.productTelemetry.enable=false
-          --set nginx.image.repository=ghcr.io/nginxinc/nginx-gateway-fabric/nginx
-          --set nginx.image.tag=${{ steps.nginx-meta.outputs.version }}
-          --set nginx.image.pullPolicy=Never
-          --set service.type=NodePort
-          -n nginx-gateway
-        working-directory: ${{ github.workspace }}/charts/nginx-gateway-fabric
+    needs: [vars, build-oss, build-plus]
+    strategy:
+      fail-fast: false
+      matrix:
+        image: [nginx, plus]
+        k8s-version: ["${{ needs.vars.outputs.min_k8s_version }}", "${{ needs.vars.outputs.k8s_latest }}"]
+    uses: ./.github/workflows/helm.yml
+    with:
+      image: ${{ matrix.image }}
+      k8s-version: ${{ matrix.k8s-version }}
+    secrets: inherit
+    # if: ${{ needs.vars.outputs.helm_changes == 'true' || github.event_name == 'schedule' }}
```

Comment:
Need this?

Reply:
This comment and the other one are to force the run, to test that it was working since it's not a `schedule`. I'm going to remove it now.

Note that `ct` by default only runs if there are changes to the chart, so this will skip the tests if there aren't any changes, but will still run for the nightly.

Reply:
just highlighting that the comment is still in the diff